osu.Game/Screens/TournamentShowcase/ShowcaseConfigScreen.cs

```diff
@@ -359,6 +393,8 @@ protected override void LoadComplete()
             });
 
             this.FadeInFromZero(500, Easing.OutQuint);
+
+            currentTabChanged(new ValueChangedEvent<ShowcaseConfigTab>(ShowcaseConfigTab.General, ShowcaseConfigTab.General));
```

Comment:
```suggestion
            currentTab.TriggerChange();
```

Suggested to use this method, which can call all events associated to the `currentTab` bindable, and is simpler to understand :)

Reply:
Yes, that's true. Thanks for the suggestion (